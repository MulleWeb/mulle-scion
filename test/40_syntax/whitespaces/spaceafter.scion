Zusam{% %}men!
Zusam{% %} men!
Zusam{% %}  men!
Zusam {% %}men!
Zusam {% %} men!
Zusam {% %}  men!
Zusam  {% %}men!
Zusam  {% %} men!
Zusam  {% %}  men!
