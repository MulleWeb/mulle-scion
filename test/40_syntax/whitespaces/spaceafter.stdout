Zusammen!
Zusam men!
Zusam  men!
Zusam men!
Zusam  men!
Zusam   men!
Zusam  men!
Zusam   men!
Zusam    men!
